### DISCUSSION TITLE
对于多程序混用的兼容问题（以及手机同时抢票的问题）

### DISCUSSION THREAD

**Discussion by duyueovo** [mikumifa/biliTickerBuy#187] [提问专区]
明天抢票的时候计划使用多个程序进行抢票（抢一个身份证 不同账号 不同时间 同一个人的票）
请问能够兼容么。
比如我同时使用 BHYG、biliTicketBuy两个程序进行同一个人的抢票，可以以此方式实现么（或者以多程序运行的方式增加概率？）

**Comment by mikumifa** [comment]
可以吧

**Reply by duyueovo** [reply]
两个程序抢一个人票呢 会产生问题么== 因为计划使用这个方法再增加一波概率

**Reply by mikumifa** [reply]
没
